src/models/userDB.js

```diff
@@ -0,0 +1,14 @@
+const userDB = [{
```

Comment:
You created a new userDB instead of using the existing one.
Also you put it in the models folder. This is some dummy data, it should be in the service.
Models don't contain actual data.